cfgs/env_cfg/acrobot_swingup.yaml

```diff
@@ -0,0 +1,20 @@
+# task settings
+task: acrobot_swingup
+frame_stack: 3
```

Comment:
hm there is a lot of overlap going on, only a few arguments are changing. I wonder if there is a way to only have a task specific overrides and still preserve the default config.